helm/charts/artifacthub-repo.yml

```
@@ -0,0 +1,17 @@
+# Artifact Hub repository metadata file
+#
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: 14c17493-c338-4acf-a923-badee42eb5a0
+owners:
+  - name: Caleb Lloyd
+    email: caleb@nats.io
```

ArtifactHub has sign-in with Google.  I think to claim this repo it's got to be my `synadia` address.  You might as well add yourself there too @bruth 

```suggestion
    email: caleb@synadia.com
```

---

@wallyqs Do you have a login?

---

have a login now using `wally@synadia.com`